src/app/components/calendar/calendar.ts

```diff
@@ -2965,6 +2965,10 @@ export class Calendar implements OnInit, OnDestroy, ControlValueAccessor {
                         case 'y':
                             output += lookAhead('y') ? date.getFullYear() : (date.getFullYear() % 100 < 10 ? '0' : '') + (date.getFullYear() % 100);
                             break;
+                        case 'B':
```

Comment:
If you add a new dateFormat, you should edit the documentation and explained too :).

Thanks